docs/docs/known-issues.md

```diff
@@ -23,16 +23,21 @@ import paths. While `ksonnet` used to magically include them, Tanka follows a
 more explicit approach and requires you to install them using `jb`:
 
 ```bash
-$ jb install github.com/ksonnet/ksonnet-lib/ksonnet.beta.4/k.libsonnet
-$ jb install github.com/ksonnet/ksonnet-lib/ksonnet.beta.4/k8s.libsonnet
+$ jb install github.com/ksonnet/ksonnet-lib/ksonnet.beta.4
+$ echo 'import "ksonnet.beta.4/k.libsonnet"' > lib/k.libsonnet
 ```
 
-This installs version `beta.3` of the libraries, matching Kubernetes version
-`1.8.0`. If you need another version, take a look at
+This does 2 things:
+
+1) It installs version `beta.4` of the libraries (in vendor/ksonnet.beta.4),
+matching Kubernetes version `1.14.0`. If you need another version, take a look at
```

Comment:
I would remove the note about the version. `1.14.0` is the only pregenerated one usable at the moment. Everything else (`beta.3`, etc) is too old for recent versions and will soon be useless after all.

We plan on maintaining this lib as well and will provide more versions then. Until then, I think it is fine to just don't mention versions at all.

Reply:
We do the same in the tutorial on purpose btw

Reply:
Ok.